utils/dashboards/config.js

```diff
@@ -110,63 +110,12 @@ export default [
         categories: ['Water', 'Food']
       },
       {
-        name: '% of power plants exposed to water risk',
-        categories: ['Water', 'Energy']
-      },
-      {
         name: 'Watersheds are at risk due to deforestation',
```

Comment:
Remove the name please. If the `widgetId` is specified, the name will be loaded from the API.